### PR TITLE
Add future contributions to confirm third pillar mandate component

### PR DESCRIPTION
**Pull request by oliverviljamaa**

### CODE REVIEW

File: src/entries/components/thirdPillar/initialState.js [hunk 1]
```diff
@@ -2,5 +2,6 @@ export default {
   monthlyContribution: null,
   sourceFunds: [],
   targetFunds: [],
+  selectedFutureContributionsFundIsin: 'EE3600109419',
```
Comment (v3rm0n) [inline src/entries/components/thirdPillar/initialState.js]: What magic (number) is this?

Comment (oliverviljamaa) [reply src/entries/components/thirdPillar/initialState.js]: LHV Index Plus fund for now. How would you solve it from the API side? I was thinking about it and thought maybe one of the target funds could have a `preferred` or `recommended` flag, but I'm not too sure.

File: src/entries/components/flows/thirdPillar/ConfirmThirdPillarMandate/ConfirmThirdPillarMandate.js [hunk 2]
```diff
@@ -1,15 +1,39 @@
...
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
+import { Message } from 'retranslate';
+
+export const ConfirmThirdPillarMandate = ({ selectedFutureContributionsFund }) => (
+  <Fragment>
+    {selectedFutureContributionsFund && <Message>confirmThirdPillarMandate.intro</Message>}
+
+    {selectedFutureContributionsFund && (
+      <Fragment>
```
[the remark this answers is not in the document]
Comment (oliverviljamaa) [reply src/entries/components/flows/thirdPillar/ConfirmThirdPillarMandate/ConfirmThirdPillarMandate.js]: Good catch, will remove.

File: src/entries/components/thirdPillar/initialState.js (hunk 1 above)
Comment (oliverviljamaa) [reply src/entries/components/thirdPillar/initialState.js]: I'll extract it to a variable, though!

**Pull request merged**